src/main/scala/devices/tilelink/CLINT.scala

```diff
@@ -105,7 +105,7 @@ trait CanHavePeripheryCLINT { this: BaseSubsystem =>
   val (clintOpt, clintDomainOpt, clintTickOpt) = p(CLINTKey).map { params =>
     val tlbus = locateTLBusWrapper(p(CLINTAttachKey).slaveWhere)
     val clintDomainWrapper = tlbus.generateSynchronousDomain.suggestName("clint_domain")
-    val clint = clintDomainWrapper { LazyModule(new CLINT(params, cbus.beatBytes)) }
+    val clint = clintDomainWrapper { LazyModule(new CLINT(params, tlBusWrapperLocationMap.lift(CBUS).getOrElse(tlBusWrapperLocationMap(p(TLManagerViewpointLocated(location)))).beatBytes)) }
```

Comment:
I prefer adding a `slaveWhere` parameter to CLINTParams, rather than providing this default

Reply:
Agree. AttachParameter is required for these devices. Do you prefer add it in this PR or my following PR to clean up these usages?

Reply:
Can you do it to this PR?

Reply:
Sure! Seems only CLINT is required. (Where is PLICAttachParam?)